symbology/picture-marker-symbol/build.gradle

```diff
@@ -1,17 +1,22 @@
 plugins {
     id 'application'
-    id 'org.openjfx.javafxplugin' version '0.0.13'
+    id 'org.openjfx.javafxplugin' // version is defined in settings.gradle
 }
 
 group = 'com.esri.samples'
 
 ext {
-    arcgisVersion = '100.15.0'
+    arcgisVersion = '200.0.0-3672'
 }
 
 javafx {
     version = "17.0.2"
-    modules = [ 'javafx.controls', 'javafx.swing' ]
+    modules = [ 'javafx.controls', 'javafx.graphics', 'javafx.fxml', 'javafx.web', 'javafx.media', 'javafx.swing' ]
```

Comment:
I think we should refactor this sample to get rid of `javafx.swing` as it's a bit of an unnecessary use of it too. Looking at other samples, the key is to show:
1. creating a picture marker symbol from a URL to a service (which we do)
2. creating a PMS from an `Image` (which we do)
3. creating a PMS from a URL to a file path - which we're doing in a very convoluted way. I think we should do something like:

```
      URL res = getClass().getClassLoader().getResource("orange_symbol.png");
      File file = Paths.get(res.toURI()).toFile();
      orangeSymbol = new PictureMarkerSymbol(file.getAbsolutePath());
      placePictureMarkerSymbol(orangeSymbol, leftPoint);
```

And removing the whole bit `saveResourceToExternalStorage`.
Give me a shout if this isn't clear